main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"net/url"
+)
+
+const (
+	weatherAPI = "http://api.openweathermap.org/data/2.5/weather"
+	apiToken   = "2ae6e50490e35649304462a5d8f6cb29"
```

Comment:
It is important not to push your secrets in Github repo. Store it in .env file for example

Reply:
fixed